Formula/gh.rb

```diff
@@ -0,0 +1,27 @@
+class Gh < Formula
+  desc "GitHub command-line tool"
+  homepage "https://github.com/cli/cli"
+  url "https://github.com/cli/cli/archive/v0.5.7.tar.gz"
+  sha256 "a1185de36b27bb9831a172b70e60f741aeaa774682f3ac4bccb9df45734f8b5b"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -X github.com/cli/cli/command.Version=#{version}
+      -X github.com/cli/cli/command.BuildDate=#{Date.today}
+      -s -w
+    ]
+    system "go", "build", "-trimpath", "-ldflags", ldflags.join(" "), "-o", bin/name, "./cmd/gh"
+
+    (bash_completion/"gh").write `#{bin}/gh completion -s bash`
```

Comment:
Does the `install` block also run when puring a Homebrew bottle? (I'm assuming not). If not, which mechanism will move these completions into place?

If yes, do we need to skip the `go build` part in that case?

Reply:
It does not. It'll be handled by the bottle being extracted. If this is somehow broken (although I've not seen any evidence of this yet) then it could be moved into `post_install`.